downloads/swan-lake-release-notes/swan-lake-beta2/RELEASE_NOTE.md

```diff
@@ -1,42 +1,217 @@
+
+## Runtime Updates
+
+### Improvements
+
+- Introduced `configurable` variables of union types via the TOML syntax. 
+
+```ballerina
+configurable map<anydata> myMap = ?;
+configurable int|string id = ?;
+```
+
+**TOML:**
+
+```toml
+id = "12345"
+
+[myMap]
+name = "John"
+age = 10
+```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `postgresql` Package
+
+Introduced the new `postgresql` database package with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.
+
+#### `log` Package
+
+Added observability span context values to log messages when observability is enabled.
+
+#### `io` Package
+
+Introduced the `io:fprint` and `io:fprintln` APIs which allows the users to print content to a given stream (standard error or standard out).
+
+```ballerina
+io:fprint(io:stderr, "Unexpected error occurred");
+io:fprintln(io:stderr, "Unexpected error occurred");
+io:fprint(io:stdout, "Passed without an error");
+io:fprintln(io:stdout, "Passed without an error");
+```
+
+#### `websocket` Package
+
+Introduced the declarative auth support for the server-side. 
+
+#### `websub` Package
+
+Added a utility method to retrieve HTTP headers from the `websub:ContentDistributionMessage`.
+
+### Improvements
+
+#### Database Packages
+
+Made the return type of the stream to be inferred now as a second parameter to the query remote method.
+
+**Previous Syntax**
+
+```ballerina
+stream<record{}, error> resultStream =
+        jdbcClient->query(`SELECT * FROM Customers`, Customer);
+
+stream<Customer, sql:Error> customerStream =
+        <stream<Customer, sql:Error>>resultStream;
+```
+
+According to the previous syntax, you can give the `Customer` record as the expected record type of the stream but you can not get that stream directly. For that, you have to cast the returned stream to the expected stream. Therefore, this has been improved to directly get data as an expected record type.
+
+**New Syntax**
+
+```ballerina
+stream<Customer, error> customerStream = sqlClient->query(`SELECT * FROM Customers`);
+```
+
+#### `http` Package
+
+- Updated to respond with a 202 Accepted response when the resource function returns nil. If the `http:Caller` is used in the resource, then, returning nil from the resource leads to a 500 Internal Server Error response.
+- Updated to log the error stack trace when an error is returned from the resource function and on connection failures.
+- Updated to log a warning when the same request is responded more than one time.
+
+#### `graphql` Package
+
+- Made the GraphQL resource execution non-blocking.
```

Comment:
```suggestion
- GraphQL resource execution is now non-blocking.
```

Reply:
This was changed to make the wordings consistent with the others (i.e., to start with the past tense of the verb).